mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -97,3 +97,7 @@ func restart() error {
 	}
 	return nil
 }
+
+func init() {
+	restartCmd.Flags().BoolVarP(&skipUpdateFlag, "skip-update", "", false, "(optional) skip checking for new Meshery's container images.")
```

Comment:
It works correctly on the Docker platform, there's not a need for `skip-update` for Kubernetes in `system restart`. 
LGTM, Nice work! @alphaX86